pombola/south_africa/views.py

```diff
@@ -144,87 +144,133 @@ def get_object(self):
 
         return province
 
+    def get_ward_councillors(self, location):
+        # Look up the ward on MapIt:
+        url = 'http://mapit.code4sa.org/point/4326/{lon},{lat}?type=WD'.format(
+            lon=location.x, lat=location.y
+        )
+        r = requests.get(url)
+        mapit_json = r.json()
+        if not mapit_json:
+            return []
+        ward_id = mapit_json.values()[0]['name']
+        # Then find the ward councillor from that ward ID. There
+        # should only be one at the moment, but make it a list in case
+        # we support broader lookups in the future:
+        url_fmt = 'http://nearby.code4sa.org/councillor/ward-{ward_id}.json'
+        r = requests.get(url_fmt.format(ward_id=ward_id))
+        ward_result = r.json()
+        councillor_data = ward_result['councillor']
+        party = models.Organisation.objects.filter(
+            name__icontains=councillor_data['PartyDetail']['Name'].lower(),
+            kind__slug='party'
+        ).first()
```

Comment:
I assume it's better to pick the first if there is more than one than to use a `.get()`?